packages/docs/src/pages/tutorial/props/basic/index.mdx

```diff
@@ -3,6 +3,8 @@ title: Component Props
 layout: tutorial
 ---
 
-Web applications are built up from components in the same way that general applications are built up from functions. Composing functions would not be very useful if functions would not allow us to pass parameters to them. In the same way that functions have parameters components have props. We use props to pass data from parent to child components.
+Web applications are built up from components in the same way that general applications are built up from functions.
 
-Modify the code so that the parent `<App>` can pass `salutation="Hello"` and  `name="World"` to the `<Greeter>`. Add properties to the `GreeterProps` interface and then add the necessary bindings.
\ No newline at end of file
+Composing functions would not be very useful you couldn't pass in parameters. In the same way that functions have parameters, components have "props". A component uses props to pass data from parent to child components.
```

Comment:
missing an "if" after "be very useful"

Plus - I would modiy the last sentence to "A component uses props to pass data to its children components"

Reply:
thanks for catching that 😄